Routing.Test/Routing.Test.csproj

```diff
@@ -2,6 +2,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <NullableContextOptions>enable</NullableContextOptions>
```

Comment:
Remove this as it is no longer needed.

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/370cb929460c1039c268cf4b22e4776d4678ac27